MozillaFoundation/foundation.mozilla.org
# Update external links attributes to open in a new tab

# Description

This PR updates all external links attributes so that these links open in a new tab.

## To test

1. Open the [review app homepage](https://foundation-s-tp1-260-al-safchz.herokuapp.com/).
2. Click on the hero link cta button, it should open in a new tab.

![image](https://github.com/user-attachments/assets/bd1ff84c-31b8-4e4d-851d-a2b402da6aa9)

This behavior will be consistent in preview/staging apps, but not in production, since all preview/staging apps hero links point to production urls which are served from a different domain.

Link to sample test page: https://foundation-s-tp1-260-al-safchz.herokuapp.com/
Related PRs/issues: #8214

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1354)
